.github/ISSUE_TEMPLATE/Other.yml

```diff
@@ -0,0 +1,23 @@
+name: Other
+description: Use this if you want to add any Communities, Jobs, OSS projects or any other thing.
```

Comment:
kinda worried about spam with this issue template. Should we mention what kind of communities/ job postings are allowed?

Reply:
Hmm... every community will be evaluated by us so it will eventually reduce spam anyways

Reply:
I don't think there will be spam because, in the end, maintainers have to check and merge the prs.

Considering adding the filters as @starlightknown is saying for mentioning what kind of communities can be added, see there will be plenty of organizations from different fields, we can add the fields as suggestions according to the roadmaps given.

Reply:
@octonawish-akcodes that's a nice suggestion. Can you please add that maybe so we can have a preview of this?